### ISSUE TITLE
ccmap

### ISSUE THREAD

**Issue by pllim** [astropy/ccdproc#395]
Prompted by @saraogaz in [astropy-dev mailing list asking about ccmap equivalent](https://groups.google.com/forum/#!topic/astropy-dev/7OJSXlVWBWI), I am wondering if there is already something like that in `ccdproc`. It seems to me that this would be a logical place for such a thing to exist if it does at all.

`ccmap` documentation: http://stsdas.stsci.edu/cgi-bin/gethelp.cgi?ccmap

**Comment by crawfordsm** [comment]
CCMAP itself hasn't been on our wish list for awhile but finding astrometric solutions to an image has been.  It's been long on the to-do list although some progress has been made on it in the astroquery package (although I think that PR was never completed).  

See #160  and also https://github.com/astropy/astroquery/pull/521

**Comment by SaraOgaz** [comment]
Any updates on this?  Looks like the linked stuff hasn't been touched in a while.  I wanted to double check there weren't some other solutions in other places...

**Comment by mwcraig** [comment]
Short answer, no. I'd *really* like astrometry.net to work in astroquery, so if I have time this week I'll work on that.

**Comment by bsipocz** [comment]
Hmm, that PR in astroquery is very much abandoned. However, if anyone is keen on picking it up, it shouldn't be too far from being finished.

**Comment by mwcraig** [comment]
Yeah, I've had it open in a browser tab for ~1.5 months. Looks very close to done.
